interface/ceed-operator.c

```diff
@@ -758,14 +802,14 @@ int CeedOperatorSetField(CeedOperator op, const char *field_name, CeedElemRestri
     op->has_restriction = true;  // Restriction set, but num_elem may be 0
   }
   CeedCall(CeedBasisReferenceCopy(b, &(*op_field)->basis));
-  if (op->num_qpts == 0) op->num_qpts = num_qpts;
+  if (op->num_qpts == 0 && !op->is_at_points) op->num_qpts = num_qpts;  // no consistent number of qpts for OperatorAtPoints
```

Comment:
Note (for somewhere around here) - for an AtPoints operator, only a AtPoints restriction can be CEED_BASIS_NONE + CEED_EVAL_NONE